### ISSUE TITLE
Rollback or update

### ISSUE THREAD

**Issue by ChrisRG711** [GitHubJens/jensandersson.blog#1]
Greetings, 

Thanks in advance for any feedback. I have implemented this in our Azure sub. (Fantastic stuff btw)  After implementing it seemed as though some of the machines were not obeying and it was skipping them. The log output said that they were being skipped because the ignoreshutdown was yes. But the tag on the VM said no.  

i tried rounds of messing with it myself but alas not doing it. As part of the attempts i think i implemented tagging at different levels where the ignoreshutdown tag was on everything (nics, disks etc)  Not sure if that would affect this as it looks like it only looks at the VM tag.  The only thing i can figure is that it sees the tag name ignoreshutdown and for some reason bombs on looking at the value?  I tried to undo everything and start over but getting rid of tags is a major pain. 

Anyu thoughts?
![ignore](https://user-images.githubusercontent.com/9805053/42525510-53c59f38-8441-11e8-9118-46eff0de8ab5.PNG)

**Comment by GitHubJens** [comment]
Yup, think I forgot to initialize the variable so it would follow to the next VM in the iteration. I Will fix it in the script.

![image](https://user-images.githubusercontent.com/30659197/49407210-1f5a6280-f758-11e8-9e6d-5f7d63817a3c.png)
